pkg/virt-config/featuregate/active.go

```diff
@@ -128,6 +128,12 @@ const (
 	//
 	// PasstIPStackMigration enables seamless migration with passt network binding.
 	PasstIPStackMigration = "PasstIPStackMigration"
+
+	// Owner: @DariuszPorowski
```

Comment:
not sure if this should be assigned to my account?

Reply:
We'll need to figure out who exactly will own this feature before merging into upstream. Added an issue to track this: https://github.com/kaizentm/kubevirt/issues/24

Reply:
yep, maybe we can use general @microsoft org name. For now, I will keep my name until we decide the final one